build/config.props

```diff
@@ -32,18 +32,16 @@
     <VsTargetMajorVersion>$([MSBuild]::Add(11, $(MajorNuGetVersion)))</VsTargetMajorVersion>
     <VsTargetBranch>master</VsTargetBranch>
     <VsTargetChannel>int.$(VsTargetBranch)</VsTargetChannel>
+    <VsTargetBranch Condition="'$(IsEscrowMode)' == 'true'">rel/d$(VsTargetMajorVersion).$(MinorNuGetVersion)</VsTargetBranch>
+    <VsTargetChannel Condition="'$(IsEscrowMode)' == 'true'">int.d$(VsTargetMajorVersion).$(MinorNuGetVersion)</VsTargetChannel>
+
     <!-- when LockSDKVersion is true,  it will use the specific version in CliVersionForBuilding for building, and specific version in OverrideCliBranchForTesting for testing,  -->
     <!-- when LockSDKVersion is not true, it will ignore the properties and just use the latest version for the channel specified in CliVersionForBuilding for building, CliBranchForTesting for testing -->
-    <LockSDKVersion>true</LockSDKVersion>
+    <LockSDKVersion></LockSDKVersion>
```

Comment:
For clarity, how about:
```XML
<LockSDKVersion>false</LockSDKVersion>
```

Reply:
Thanks Damon.
I searched "Condition=" in our code, and found most of them are comparing with 'true' or empty string
E.g. https://github.com/NuGet/NuGet.Client/blob/dev/build/build.proj#L357-L396
So I guess this is our code style?

Reply:
https://github.com/NuGet/NuGet.Client/blob/dev/build/config.props#L29

https://github.com/NuGet/NuGet.Client/pull/3162/files#diff-46b8b89f0b5ffc7c8be0713918765ce6R16-R21

Reply:
I see. Changed. Thanks.